pkgs/by-name/ed/edmarketconnector/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  python3,
+  makeWrapper,
+}:
+let
+  pythonEnv = python3.buildEnv.override {
+    extraLibs = with python3.pkgs; [
+      tkinter
+      requests
+      pillow
+      (watchdog.overrideAttrs {
+        disabledTests = [
+          "test_select_fd"
+        ];
+      })
+      semantic-version
+      psutil
+    ];
+    ignoreCollisions = true;
+  };
+in
+stdenv.mkDerivation rec {
```

Comment:
Using `finalAttrs` might be better to allow propagation of an overriden `src` attribute in installPhase.